lists/US-SAM

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "System for Award Management",
```

Comment:
Thanks @kd-ods - maybe this could serve as an example of how to do half a job for @HollyODSC !

Anyway, I've moved and change the filename and updated the name.